src/components/DistanceEReceipt.tsx

```diff
@@ -29,9 +30,14 @@ type DistanceEReceiptProps = {
 
 function DistanceEReceipt({transaction, hoverPreview = false}: DistanceEReceiptProps) {
     const styles = useThemeStyles();
-    const {translate} = useLocalize();
+    const {translate, preferredLocale} = useLocalize();
     const thumbnail = hasReceipt(transaction) ? getThumbnailAndImageURIs(transaction).thumbnail : null;
-    const {amount: transactionAmount, currency: transactionCurrency, merchant: transactionMerchant, created: transactionDate} = getTransactionDetails(transaction) ?? {};
+    const {
+        amount: transactionAmount,
+        currency: transactionCurrency,
+        merchant: transactionMerchant,
+        created: transactionDate,
+    } = getTransactionDetails(transaction, CONST.DATE.FNS_FORMAT_STRING, undefined, preferredLocale) ?? {};
```

Comment:
This is used for display only, no? So why would it use `FNS_FORMAT_STRING` ( 'yyyy-MM-dd`) instead of a locale aware date?